misc/host_tests.conf

```diff
@@ -9,6 +9,7 @@ add fail
 add ping
 add bacnet
 add mudgee
+add password
```

Comment:
this should be in all_tests, not host_tests, and follow model for switch tests (include a build.conf file)